src/content/docs/alerts/get-notified/notification-integrations.mdx

```diff
@@ -740,7 +740,11 @@ You can update ServiceNow attributes directly in the target table, such as **Inc
 
   ### Prerequisites [#slack-prereqs]
 
-  Your Slack workspace needs to have the [New Relic application](https://newrelic.slack.com/apps/AP92KQJS3-new-relic?tab=more_info)(or the [EU App](https://slack.com/apps/AS5D75HQQ-new-relic-eu?tab=more_info) for `one.eu.newrelic` customers) installed. A workspace administrator must approve the application before you can individually install it. If you need assistance, contact support@newrelic.com.
+  Your Slack workspace needs to have the [New Relic application](https://newrelic.slack.com/apps/AP92KQJS3-new-relic?tab=more_info)(or the [EU App](https://slack.com/apps/AS5D75HQQ-new-relic-eu?tab=more_info) for `one.eu.newrelic` customers) installed. A workspace administrator must approve the application before you can individually install it. If you need assistance, you can get help by following these steps:
```

Comment:
How about? 

"If you need assistance, follow these steps in the UI:"

Reply:
Ah, thanks for the catch